### PR TITLE
test pull request

### PR DESCRIPTION
**Pull request by Cassiopeia316**

### CODE REVIEW

File: chapter3/devinagunawan/devinagunawan_project/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,935 @@
+{
+  "name": "devinagunawan-project",
```
Comment (aindrajaya) [inline chapter3/devinagunawan/devinagunawan_project/package-lock.json:2]: Kalau bisa yang `package-lock.json` di masukan ke `.gitignore`

**Review by aindrajaya — COMMENTED** [review]
Good, but need to solve one problem at the `.gitignore` by adding `package-lock.json`

File: chapter3/devinagunawan/devinagunawan_project/package-lock.json (hunk 1 above)
Comment (Cassiopeia316) [reply chapter3/devinagunawan/devinagunawan_project/package-lock.json:2]: sudah saya masukan ya mas

**Review by aindrajaya — COMMENTED** [review]
Done!

**Pull request closed without merging**